src/components/LoginForm.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { Link, useHistory } from 'react-router-dom'
+import { isUserValid } from '../utils/authentication'
+
+export default function LoginForm() {
+    const [error, setError] = useState('')
+    const history = useHistory()
+    const [loginDetails, setLoginDetails] = useState({ username: '', password: '' })
+
+    const submitHandler = async (e) => {
+        e.preventDefault()
+
+        try {
+            const isValid = await isUserValid(loginDetails)
+            if (isValid) {
+                history.push({ pathname: "/welcome", state: { username: loginDetails.username } })
+            } else {
+                setError('INVALID CREDENTIALS')
+            }
+        } catch (err) {
+            setError('INVALID CREDENTIALS')
+        }
+    }
+
+    return (
+        <div className="container"
+            style={{
```

Comment:
Store css into another file and access it's properties by className attribute attached to the html elements